CHANGELOG.md

```diff
@@ -17,9 +17,9 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * Updated Certbot and its plugins to improve the handling of file system permissions
   on Windows as a step towards adding proper Windows support to Certbot.
 * Updated urllib3 to 1.24.2 in certbot-auto.
-* Removed the fallback introduced with 0.32.0 in `acme` to retry a response
-  challenge with a `keyAuthorization` after a failed response without this field.
-
+* Removed the fallback introduced with 0.32.0 in `acme` to retry a challenge response
+  with a `keyAuthorization` if sending the response without this field caused a
+  `malformed` error to be received from the ACME server.
 ### Fixed
```

Comment:
nit: Looks like we lost a blank line above this.

Reply:
Oups